client/landing/gutenboarding/stepper-onboarding/hooks/use-steps.tsx

```diff
@@ -0,0 +1,5 @@
+import { Steps } from '../steps';
+
+export function useSteps() {
+	return Object.values( Steps );
```

Comment:
Great to break this out into its own hook rather than just calling `Object.values( Steps )` directly within `useStepNavigation()`

When I experimented with implementing the signup flow in Gutenboarding (pbAok1-2iM-p2#comment-4698), I found that the `useSteps()` hook provides the magic that facilitates branching. The ability to return different steps depending on the app state is what implements the finite-state-machine ideas folks were talking about in the pbAok1-2iM-p2 thread.

What you've got here is excellent; the flow is currently very linear. But I just wanted to drop a comment so we don't get too attached to the idea that `useSteps()` will always return a spread out list of declarative steps.